### DISCUSSION TITLE
init with multiple commands

### DISCUSSION THREAD

**Discussion by eadmaster** [ajeetdsouza/zoxide#709] [Q&A]
is it possible to init bash with multiple command aliases?
e.g.
````
eval "$(zoxide init bash --cmd z jump j cd)"
````

**Comment by 0rphee** [comment]
+1 on this. I would like to alias it to cd and j in the same command.

**Comment by morenathan** [comment]
That could be nice, so adding a +1. Although I'm a bash user, it'd be great to overwrite CD. It would be even better to have those multiple cmd's be more useful, like aliases to compound jump+run.

Ex.
e (partial+space+tab)/somefile (result: opens somefile in your editor)

TBH: I haven't dug into anymore of the mechanics of zoxide to see if this is possible. On the surface it doesn't seem to be.

**Comment by ajeetdsouza** [comment]
This could also just be a user-defined alias, right?
